warp/warp.cabal

```diff
@@ -114,7 +114,7 @@ library
         stm >=2.3,
         streaming-commons >=0.1.10,
         text,
-        time-manager >=0.2 && <0.3,
+        time-manager >=0.2 && <0.4,
```

Comment:
If we change the API, then we'll need to adjust `warp` as such as well.

If we support both `time-manager 0.2 && 0.3`, which is something we should do, then we'll need to adjust `withII` in `Network.Wai.Handler.Warp.Run`. But not much more, I expect.

Reply:
In warp, most `pause ` and `resume` are used correctly as a pair.

Unfortunately,  `pause` is used solely `timeoutBody`.
We should understand cd82d85b060b720e2cd89b035b2a101b943c8273.